azure-pipelines.yml

```diff
@@ -227,23 +249,38 @@ steps:
         kubectl logs -n $(OPERATOR_NAMESPACE) deployment/azureoperator-controller-manager -c manager
  
   ## TODO: Add a Health check task to ensure operator deployed to AKS works
+  
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: 'ASO Subscription'
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        echo "Chosen AKS Cluster name"
+        echo $(chosenclustername)
```

Comment:
Similar to a previous comment but you can use an env block to show that you're passing in a variable from the previous task. That way if the output name changes in the previous task I think it will break this task a little more explicitly.

Reply:
I experimented with that a bit but it looks like it works only for jobs.